src/hooks/hookResources.ts

```diff
@@ -410,12 +420,27 @@ const getCardOrEmptyArrayFromRules =
     drug: MongooseMedication | null,
     remsCase: RemsCase | null,
     request: MedicationRequest,
-    patient: Patient | undefined
+    patient: Patient | undefined,
+    pharmacy: HealthcareService | undefined
   ) =>
-  (rule: CardRule): Card | never[] => {
+  async (rule: CardRule): Promise<Card | never[]> => {
+    let pharmacyInfo = '';
+    console.log('Checking pharmacy certification for ' + pharmacy);
+    if (pharmacy) {
+      const isCertified = await checkPharmacyCertification(pharmacy, drug?.code); // AWAIT HERE
+
+      const pharmacyName = pharmacy.name || 'Selected pharmacy';
+
+      pharmacyInfo = `**Pharmacy Status:** ${pharmacyName} is ${
+        isCertified ? 'certified' : 'not yet certified'
+      } for ${display} REMS dispensing. This medication ${
+        isCertified ? 'can' : 'cannot yet'
+      } be dispensed at this location.\n\n`;
+    }
+
     const card = new Card(
       rule.summary || display || 'Rems',
-      rule.cardDetails || CARD_DETAILS,
+      pharmacyInfo + (rule.cardDetails || CARD_DETAILS),
```

Comment:
Do we want to add the details here about the pharmacy? Or should it be a separate card? Is "Test Pharmacy" the pharmacy name? We should probably add the location as well if that can be added to the HealthcareService.

Reply:
in the IG you mentioned putting it in the card details, we can make another card if we want too but it'll be pretty bare bones with just some text

Reply:
Haha, really? I don't remember what I put in the IG, it is fine here as well, I was just imagining a separate card since it would be a card for patient, provider, and pharmacy. This is less important than my other comments and something we could update later on if we wanted.